src/specs/api.yaml

```diff
@@ -458,16 +458,11 @@ components:
           type: string
           example: GetEmbedding
           description: The name of this work -- must be `GetEmbedding`.
-        type:
-          type: string
-          enum:
-            - pca
-            - tsne
-            - umap
-          description: The type of embedding to compute.
+        config:
+          type: object
+          description: The configuration settings to use for this embedding. This will be used both to configure the embedding and to cache the results.
```

Comment:
I would rephrase this description as `The configuration settings to use for calculation of this embedding.` I wouldn't have the `This will be used both to configure the embedding and to cache the results.` part because the caching applies to all the tasks, not only to the embedding